src/test/java/SetTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트 구현")
+    @Test
+    void set_메소드_학습테스트() {
+        int actual = numbers.size();
+
+        assertThat(actual).isEqualTo(3);
```

Comment:
다음과 같이도 단언문을 사용할 수 있습니다. ㅎㅎ
```suggestion
        assertThat(actual).hasSize(3);
```